extensions/standard-processors/processors/GetFile.cpp

```diff
@@ -234,15 +235,15 @@ bool GetFile::fileMatchesRequestCriteria(std::string fullName, std::string name,
 
 #ifdef WIN32
   struct _stat64 statbuf;
-  if (_stat64(fullName.c_str(), &statbuf) != 0) {
-    return false;
-  }
+  const auto stat_result = _stat64(fullName.c_str(), &statbuf);
```

Comment:
Couldnt we use https://en.cppreference.com/w/cpp/filesystem/file_size and https://en.cppreference.com/w/cpp/filesystem/last_write_time instead of stat?

Reply:
We certainly can. Updated in [4671175](https://github.com/apache/nifi-minifi-cpp/pull/1311/commits/467117511fee9b25413a0091f2cb66c579e8ecb0)